routes/adminRoutes.js

```diff
@@ -59,8 +58,34 @@ router.delete('/team/:teamId', async (req, res) => {
   }
 });
 
-router.delete('/curler/:curlerId', async (req, res) => {
+router.put('/team/:teamId', async (req, res) => {
+  try {
+    const id = req.params.teamId;
+    const { name, affiliation, note } = req.body;
+    Exceptions.throwIfNull({ id, name });
+    let success = await curlingEventService.updateTeam(id, name, affiliation, note);
+    res.status(200).send(success);
+  }
+  catch (error) {
+    console.error(error.message);
+    res.status(400).send({ error, message: error.message });
+  }
+});
 
+router.post('/team/', async (req, res) => {
+  try {
+    let { name, affiliation, note } = req.body;
```

Comment:
sorry, just found a bug. Please use identical field names from the api endpoints doc and not from the table. They may not necessarily be the same. orgId instead of affiliation